flybywiresim/aircraft
# fix(mcdu): Hide "POSITION UPDATE AT" when ADIRS not aligned

Fixes #5706 

## Summary of Changes

This PR hides the POSITION UPDATE AT field on the PROG page, if the ADIRS are not aligned, i.e when there is no GPS or inertial guidance available.

Note that I am not sure the logic is completely correct as it is. I believe the POSITION UPDATE AT field should stay hidden even if the airplane's GPS fails since it will revert to inertial guidance. I don't believe we simulate this yet, so this should suffice. 

## Screenshots (if necessary)


Before alignment:
![mcdu-before-align](https://user-images.githubusercontent.com/22713769/139137986-a655f903-bf11-4888-a791-8915894b5933.png)

After alignment
![mcdu-after-align](https://user-images.githubusercontent.com/22713769/139137984-4a308620-bb09-4c20-a539-9cf94b5c6bfc.png)

## References


- From the FCOM (thank you tracer)
![fcom](https://user-images.githubusercontent.com/9995998/130519731-2b5f0273-4a22-413e-b226-bbfa23bea643.png)
- From https://www.youtube.com/watch?v=efQlfF87c8c#t=3m48s (thank you BehEh)
![](https://user-images.githubusercontent.com/929769/130628845-31c8ade6-40fe-4c65-8e62-f5dabb3085be.png)




## Additional context



Discord username (if different from GitHub): BlueberryKing#6641

## Testing instructions

- Start cold and dark
- Power up airplane, do not align ADIRS
- Check PROG page, make sure the POSITION UPDATE AT field is visible.
- Align ADIRS
- Make sure POSITION UPDATE AT is no longer visible


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
